Updated the ProviderSdk and chocolatey.lib

Megasware128 opened the pull request:
Updated the Provider SDK to latest git revision (commit: [be20cd7](https://github.com/OneGet/ProviderSdk/commit/be20cd77b6d04c6909b05f034d622f2529c6c48b))
Updated Chocolatey Core to latest NuGet version (version: [0.9.10-beta-20160402)](https://www.nuget.org/packages/chocolatey.lib/0.9.10-beta-20160402)

ferventcoder commented on install-provider.ps1:
```diff
@@ -1,69 +0,0 @@
-
-cd $orig
-
-
```
Any reason this was deleted?

Megasware128 replied:
I deleted it since the script was broken because of the OneGet to PackageManagement renaming, I started importing the provider manually with Import-PackageProvider. But I might as well just update the script ;)

Megasware128 commented:
I added the install-provider.ps1 back and made it work again. I also I renamed the assembly back to ChocolateyProvider because somewhere while merging the new provider sdk it seems it got renamed to SampleProvider.

ferventcoder commented on ChocolateyProvider.csproj:
```diff
@@ -64,7 +64,7 @@
   <ItemGroup>
     <Reference Include="chocolatey, Version=0.9.10.0, Culture=neutral, PublicKeyToken=79d02ea9cad655eb, processorArchitecture=MSIL">
-      <HintPath>packages\chocolatey.lib.0.9.10-beta-20160411\lib\chocolatey.dll</HintPath>
+      <HintPath>packages\chocolatey.lib.0.9.10-beta-20160402\lib\chocolatey.dll</HintPath>
```
Yes, had a change that required it. I was able to fix up choco.exe prior to approval, but NuGet immediately publishes packages.
